tests/UI/pages/BO/shopParameters/stores/index.js

```diff
@@ -181,6 +200,120 @@ class Stores extends BOBasePage {
   getStoreStatus(page, row) {
     return this.elementVisible(page, `${this.tableColumnStatus(row)} a.action-enabled`, 1000);
   }
+
+  /**
+   * Set store status by clicking on column status
+   * @param page
+   * @param row
+   * @param wantedStatus
+   * @return {Promise<void>}
+   */
+  async setStoreStatus(page, row, wantedStatus) {
+    const actualStatus = await this.getStoreStatus(page, row);
+
+    if (actualStatus !== wantedStatus) {
+      await this.clickAndWaitForNavigation(page, `${this.tableColumnStatus(row)} a`);
```

Comment:
Shouldn't that be its own selector ?

Reply:
`tableColumnStatus` was not used without `a` so i updated it.
Thanks